gateway/cli/src/main.rs

```diff
@@ -91,9 +96,54 @@ pub enum Commands {
 
         #[clap(long)]
         network: Option<bitcoin::Network>,
+
+        /// Format <federation id>,<base msat>,<proportional to millionths part>
+        #[clap(long)]
+        per_federation_routing_fees: Option<Vec<FederationIdAndRoutingFeesParameter>>,
     },
 }
 
+#[derive(Clone)]
+pub struct FederationIdAndRoutingFeesParameter {
```

Comment:
See above comment....but if we elect to keep it I think `PerFederationRoutingFees` is a better name